src/util/scrollbar.spec.ts

```diff
@@ -0,0 +1,74 @@
+import {ScrollBar} from './scrollbar';
+import {noop} from 'rxjs';
```

Comment:
We probably should add `noop` in ng-bootstrap's 'utils.ts' instead of relying on the external one fow this

Reply:
The `noop` is there, as part of that PR i've removed this as the `rxjs` ships one. Would amned this for now